src/transaction/boot_sr.c

```diff
@@ -2334,6 +2334,15 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
       goto error;
     }
 
+#if defined(SERVER_MODE)
+  /* for passive transaction server, log initialization occurs earlier on because it does not rely
+   * on any type of storage; information is manually/explicitly retrieved from page server */
+  if (is_passive_transaction_server ())
+    {
+      log_initialize_passive_tran_server (thread_p);
```

Comment:
On PTS, page buffer highest evicted LSA must be set before starting to read pages.
```suggestion
      log_initialize_passive_tran_server (thread_p);
      pgbuf_highest_evicted_lsa_init ();
```

Reply:
It is already done sometime later? Or, for PTS, must the call be moved earlier, just after log initialize?

Reply:
Actually, for both type of transaction servers it should be done before fetching data pages from page server, and after log is initialized. The current active transaction server booting sequence does not allow it, because pages are fetched before the log is initialized. If page server replication remains behind, it could be a problem; fortunately, it is most likely that the page server replication finishes up by the time ATS is restarted.

On PTS however it is mandatory to make sure that the PS replication is not behind the PTS replication while fetching data pages. Therefore, having the highest evicted LSA, used as target for PS replication, is more important.

Later on, the booting sequence of ATS could be changed to become similar to the PTS.

Reply:
I added the call for evicted LSA initialization on possive transaction server and also guarded the existing call for !passive_tran_server.